internal/fnruntime/runner.go

```diff
@@ -179,32 +179,35 @@ type FunctionRunner struct {
 
 func (fr *FunctionRunner) Filter(input []*yaml.RNode) (output []*yaml.RNode, err error) {
 	pr := printer.FromContextOrDie(fr.ctx)
+	var fnOutput string
```

Comment:
I suggest using the [`strings.Builder`](https://pkg.go.dev/strings#Builder) here instead of string concat.

Reply:
done! Thanks